tensorflow_addons/optimizers/cocob.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+COntinuos COin Betting (COCOB) Backprop optimizer
+"""
+
+from typeguard import typechecked
+from tensorflow.python.ops import state_ops
+from tensorflow.python.ops import control_flow_ops
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+class COCOB(tf.keras.optimizers.Optimizer):
+    """
```

Comment:
Please be consistent with the docstring [style guide](https://www.python.org/dev/peps/pep-0257/)

Reply:
Done.

Reply:
I used the below format from the documentation.
```python
def bar():
    """
    A multi-line
    docstring.
    """
```
Ref.: https://www.python.org/dev/peps/pep-0257/ (Just above "References and Footnotes")